ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/api/CommandDeduplicationConverter.scala

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml
+package ledger.participant.state.kvutils.api
+
+import java.time.Duration
+
+import scala.concurrent.Future
+
+trait CommandDeduplicationConverter {
```

Comment:
Is it used anywhere?

Reply:
Good catch, this is actually part of a follow-up PR, should've never been included here. Deleted it